app/src/main/java/com/github/h3lp3rs/h3lp/locationmanager/GeneralLocationManager.kt

```diff
@@ -0,0 +1,32 @@
+package com.github.h3lp3rs.h3lp.locationmanager
+
+object GeneralLocationManager {
+    // locationManager contains the currently used location manager
+    private var locationManager: LocationManagerInterface? = null
```

Comment:
Maybe use lateint?

Reply:
In a normal use case locationManager isn't necessarily set before get is called so I'm don't think a lateinit would work here (without checks on initialisation which would defeat its purpose).